app/controllers/test/components/mentoring/mentor_inboxes/conversations_controller.rb

```diff
@@ -48,6 +48,12 @@ def index
     ]
 
     results = results.select { |c| c[:trackId] == params[:track].to_i } if params[:track].present?
+    if params[:filter].present?
+      results = results.select do |c|
+        c[:exerciseTitle].downcase.include?(params[:filter].downcase) ||
```

Comment:
FYI, you might want to have done something like:

```ruby
parts = params[:filter].downcase.split(' ').map(&:strip)

parts.all? do |part|
  c[:exerciseTitle].downcase.include?(part) || c[:menteeHandle].downcase.include?(part)
end
```

Which is more in line with how people "think" search works.

Reply:
I've fixed this in #78